tests/dragonfly/seeder/script-generate.lua

```diff
@@ -85,6 +88,7 @@ while true do
     if counter % 10 == 0 then
         -- calculate intensity (not normalized probabilities)
         -- please see attached plots in PR to undertand convergence
+        -- https://github.com/dragonflydb/dragonfly/pull/2556
```

Comment:
took me a few seconds to find this, now we can jump straight to the pr if needed (an it will for anyone who plays around this code)